lib/cdo/shared_constants.rb

```diff
@@ -105,6 +105,7 @@ module SharedConstants
     poetry_hoc
     thebadguys
     music
+    pythonlab
```

Comment:
Intentional change?

Reply:
Yes, when I added 'AppName' as a dimension, I realized that 'pythonlab' had not yet been added to this yet. I confirmed with @samantha-code that this is okay and tagged Molly in the PR description. Thanks!